### ISSUE TITLE
Map Coordinate Extents too large

### ISSUE THREAD

**Issue by LonelySauce** [BEEmod/BEE2.4#1571]
Here's a video I recorded explaining the issue (1080p recommended): https://youtu.be/_TzpHyC_0iA
I did a quick search and did not find a solution to this problem nor an issue that was open or closed. I am running PURELY VANILLA PORTAL 2 with no addons or mods other than BeeMod. I have plenty of Memory and Processing power on both ends dedicated to Portal 2.

The map is at maximum length but not width or height as shown in the video above ^
This probably isn't a bug with BeeMod but I just wanted to be double sure since I've made max-size maps before on an older PC in Vanilla portal and it worked just fine.

TL;DR: I don't know what's causing this because this same chamber was working perfectly fine before on the same date. Plz Fix.
Edit: Grammar Improvements

**Comment by LonelySauce** [comment]
Alright I fixed the bug which was way easier than I thought. Turns out the sphere catcher pit thingy causes your game to throw out that error showed in the video. I recommend you get to fixing that ASAP

**Comment by Luke18033** [comment]
Doesn't seem to happen anymore, could you send the map file?

**Comment by Luke18033** [comment]
Going to assume this is fixed, open a new issue if it still happens.
